src/Parser/Comment.php

```diff
@@ -66,29 +66,28 @@ public function parse(): Result
 
         $finalValidations = $this->commentStrategy->endOfLoopValidations($this->lexer);
 
-        $this->warnings = array_merge($this->warnings, $this->commentStrategy->getWarnings());
+        $this->warnings = [...$this->warnings, ...$this->commentStrategy->getWarnings()];
 
         return $finalValidations;
     }
 
 
     /**
-     * @return bool
+     * @return void
      */
-    private function warnEscaping(): bool
+    private function warnEscaping(): void
```

Comment:
I like the change.
Changing this, we should find a better way for the implementation I believe, avoiding empty returns for a void method.

Reply:
Not sure if we have to avoid empty returns to be honest.
A `void` function is basically a function that doesn't have any kind of return value. The 2 returns in this function are only there to short-circuit the code, nothing else.
Not returning anything is perfectly fine in this scenario - especially because it's an internal method, only being used in one place without storing its return value anyway.

Thoughts?

Reply:
I agree, I'm not looking at changing to return something specific.
My thoughts are more around those 2 short-circuits and if there is a way to improve what is being done.
Would it be more clear an `if-if else-else` statement rather than 2 separate ifs?

If nothing comes up, I'm fine with it, just wanted to have your ideas on it.

Reply:
I think the current solution is just fine, functional, and readable. My best idea would be merging the 2 ifs into one, but considering the length of the conditions, it might be better as-is. Resolving this for now, but I'm not against changes if this comes up later again.